enterprise/frontend/src/metabase-enterprise/database_routing/DestinationDatabasesList/DestinationDatabasesList.tsx

```diff
@@ -61,7 +61,7 @@ export const DestinationDatabasesList = ({
                 td="underline"
                 to={Urls.viewDestinationDatabases(primaryDatabaseId)}
               >
-                View all {destinationDatabases.length}
+                {t`View all ${destinationDatabases.length}`}
```

Comment:
This needs a context string that explains what the variable is and means. The translator should be told that the variable describes databases